### ISSUE TITLE
feat(patch): more subtitle sizes / os independent caption settings

### ISSUE THREAD

**Issue by castdrian** [revanced/revanced-patches#1888]
### Application

YouTube

### Issue

Currently youtube makes use of the OS settings to define caption style and size. However the OS defaults are only 3 sizes which results in too big or too small subtitles depending on device and OS version

### Patch

Ideally OS independent subtitle settings, for at least the size of it,  because the OS predefined size options are rendered either too big or too small on my device, maybe a pinch to zoom alongside the functionality that was requested in another open issue, namely the one that requests moving subtitles around by dragging them

### Motivation

The Android default size settings aren't refined enough to achieve a proper resizing/can't be set per-app so every application is using the same caption size and style unless it is designed to use its own caption settings

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by taskhawk** [comment]
This would help Samsung Galaxy Fold owners a lot.

See:
https://www.reddit.com/r/GalaxyFold/comments/nbjspt/captions_subtitles_are_huge_on_galaxy_z_fold_2/

**Comment by castdrian** [comment]
yep, I have a fold 4 which is the reason I opened this issue to begin with
